### ISSUE TITLE
Thank you page: First name "is a required value .

### ISSUE THREAD

**Issue by stefanliden** [Billmate/magento-2-billmate-checkout#103]
Customer tested **growme-m2** pull request for handling the thank you page.

Customer made 1-3 best choices and at 3 it went wrong.

You can see what went wrong below in M2

[2020-10-29 19:43:11] main.ERROR: Array
(
    [date] => 2020-10-29 19:43:11
    [0] => Could not create order
    [__FILE__] => /var/www/zorro.utvsite.se/app/code/Billmate/BillmateCheckout/Model/Order.php
    [__CLASS__] => Billmate\BillmateCheckout\Model\Order
    [__FUNCTION__] => create
    [__LINE__] => 141
    [exception.message] => "Förnamn" är ett obligatoriskt värde.
"Efternamn" är ett obligatoriskt värde.
    [exception.file] => /var/www/zorro.utvsite.se/vendor/magento/module-customer/Model/ResourceModel/Customer.php
    [exception.line] => 193
)
 [] []

**Comment by MattiasNilsson** [comment]
```
[2020-11-05 06:32:15] main.ERROR: Array
(
  [date] => 2020-11-05 06:32:15
  [0] => Could not create order
  [__FILE__] => /var/www/zorro.utvsite.se/app/code/Billmate/BillmateCheckout/Model/Order.php
  [__CLASS__] => Billmate\BillmateCheckout\Model\Order
  [__FUNCTION__] => create
  [__LINE__] => 130
  [exception.message] => No such entity with cartId = 
  [exception.file] => /var/www/zorro.utvsite.se/vendor/magento/framework/Exception/NoSuchEntityException.php
  [exception.line] => 51
)
```

**Comment by MattiasNilsson** [comment]
Solved with #114
